lib/builders/userGroups/userGroupBuilder.ts

```diff
@@ -74,7 +73,7 @@ export class UserGroupBuilder {
 
   addPermission() {
     const builder = new UserGroupPermissionBuilder(this);
-    this.userGroupPermissionBuilders.push(builder);
+    this.userGroupPermissionBuilders = builder;
```

Comment:
```suggestion
    this.userGroupPermissionBuilder = builder;
```
Why was this changed? 😮

Reply:
Because this.userGroupPermissionBuilder.build() already returns an array.